frontend_vue/src/components/ModalToken.vue

```diff
@@ -304,4 +304,12 @@ watch(isLoading, () => {
     }, 2000);
   }
 });
+// Only shoot confetti after generating a new token
+watch(modalType, (newVal, oldVal) => {
+  if (oldVal === 'addToken' && newVal === 'newToken') {
```

Comment:
may we replace these hardcoded strings with the enum ModalType?
`  if (oldVal === ModalType.AddToken && newVal === ModalType.NewToken) {
`

Reply:
Oh yeah, totally